src/main/resources/messages.properties

```diff
@@ -27,6 +27,7 @@ error.invalid_duration=Invalid duration specified. The supported duration values
 error.invalid_json_key=The request includes an unrecognized parameter ''{0}''. Please update or remove this parameter and retry your request.
 error.invalid_key_length=The provided key length is not supported. Valid values include '2048', '3072' and '4096'.
 error.invalid_token_signature=The request token signature could not be verified. Please validate that your request token was issued by the UAA server authorized by CredHub.
+error.invalid_key_format=The provided key format is not supported.
```

Comment:
It would probably be nice to say which formats are supported to help the user do "the right thing"